client/src/components/layouts/default.tsx

```diff
@@ -213,7 +213,7 @@ class DefaultLayout extends Component<DefaultLayoutProps> {
               removeFlashMessage={removeFlashMessage}
             />
           ) : null}
-          {children}
+          {fetchState.complete && children}
```

Comment:
What problem is this solving?

Reply:
The issue is that default pages, including https://www.freecodecamp.org/, will display incorrectly if the api fails to respond:

![image](https://user-images.githubusercontent.com/15801806/156329654-f54d7dcf-cb7c-4750-84c2-61c662a4cc5f.png)

Let me know if you want to discuss/pair.

Reply:
I'm not convinced there's a problem here. Wouldn't pretty much everything display incorrectly if the api fails to respond? This checks if fetchState is complete - which would be true or false. Whether it finds a user or not, it goes to true. I would be willing to pair up if you still think there's an issue here.